README.md

```diff
@@ -1,5 +1,5 @@
-[![VFX Platform](https://img.shields.io/badge/vfxplatform-2020-blue.svg)](http://www.vfxplatform.com/)
-[![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/)
+[![VFX Platform](https://img.shields.io/badge/vfxplatform-2023-blue.svg)](http://www.vfxplatform.com/)
```

Comment:
Just like for Python versions, we should say we support multiple VFX versions: CY2023	CY2022	CY2021	CY2020